resources/views/contact/contact_form.blade.php

```diff
@@ -11,10 +11,14 @@
                             <form action="/contact" method="POST">
                                 @csrf
                                 <h1 class="mb-4">Contact Us</h1>
-                                @if (session()->get('success'))
+                                @if (session()->get('success') === true)
```

Comment:
Laravel injects the variable to your view file so you can do this instead of accessing the session directly:
```suggestion
                                @if (isset($success) && $success === true)
```